src/constants/methods.ts

```diff
@@ -39,6 +40,8 @@ export const insertSigningAddress = (
             {
               ...exampleParams.params[0],
               from: parsedAddress,
+              to: '0xE7d522230eFf653Bb0a9B4385F0be0815420Dd98', // safe recovery address in case funds are accidentally sent
```

Comment:
oooh smart

Reply:
who address this?

Reply:
my test wallet. not sure if there is a better shared one that isn't compromised since former employees might have access...

Reply:
ehh, works for me